### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by oguzbilgener**
There were a few instances of `exgesis`, fixed them. `ExegesisCompiledOptions` is not an exported type so this shouldn't be a breaking change.

### CODE REVIEW

File: src/oas3/index.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import oas3 from 'openapi3-ts';
 
 import OpenApi from './OpenApi';
-import { ExgesisCompiledOptions } from '../options';
+import { ExegesisCompiledOptions } from '../options';
+
+export { OpenApi };
```
Comment (oguzbilgener) [inline src/oas3/index.ts]: Oh oops. I'm going to make use of this in another PR...

**Comment by jwalton** [comment]
Thanks for fixing my typing.  :)
